server/app/services/export/QuestionJsonPresenter.java

```diff
@@ -266,7 +270,11 @@ class SingleSelectJsonPresenter implements QuestionJsonPresenter<SingleSelectQue
     @Override
     public ImmutableMap<Path, String> getJsonEntries(SingleSelectQuestion question) {
       return ImmutableMap.of(
-          question.getApplicantQuestion().getContextualizedPath().join(Scalar.SELECTION),
+          question
+              .getApplicantQuestion()
+              .getContextualizedPath()
+              .join(Scalar.SELECTION)
+              .asApplicationPath(),
```

Comment:
This covers `QuestionType.DROPDOWN` and `QuestionType.RADIO_BUTTON` removed in the set.